### DISCUSSION TITLE
GPU error

### DISCUSSION THREAD

**Discussion by Duwimpuhwe** [mrdbourke/pytorch-deep-learning#480] [Q&A]
device="cuda" if torch.cuda.is_available else "cpu"

# Create an instance of model_1
torch.manual_seed(42)
model_1=FashionMNISTModelV1(input_shape=784, #this the out of the flatten after our 28*28 image goes in
                            hidden_units=10,
                            output_shape=10).to(device) # Send to the GPU if it's available
model_1

Error

RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

how can I fix this?

**Comment by IIsameerII** [comment]
Could you give more information on where you are running (colab or in your hardware).

If you are running in colab, It might help to check whether you have switched to GPU settings.

<img width="754" alt="image" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/88422517/1cee2d57-1742-46d2-8cfc-6e04ba79fcc8">

on your tab select runtime --> change runtime type and select GPU.

**Reply by Duwimpuhwe** [reply]
![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/112572927/0da3b4e1-d3ac-4145-9322-c730b7a74953)

**Reply by Duwimpuhwe** [reply]
the run type is changed but the issue is still open.
